freqtrade/freqtradebot.py

```diff
@@ -922,6 +922,11 @@ def execute_entry(
         ):
             logger.info(f"User denied entry for {pair}.")
             return False
+
+        if trade and self.handle_similar_open_order(trade, side, amount):
```

Comment:
I find the choice of placing this interesting (probably not where i'd have placed this instinctively) - but i'm not sure it's the correct place (it'll only protect entries, not exits) - at least not without a further call from the exit branch.

`execute_entry` is called 3 times (ignoring RPC and the duplicate call from `create_trade()`).
This scenario can't happen from `create_trade()` (we don't act on entry signals if an order is open).
it's also irrelevant from `replace_order()` - so the only place this CAN trigger is from `check_and_call_adjust_trade_position()`.

On the other side - this is the first place where we know the price for an entry order (which is currently ignored - but i think should be factored in?)

I'm not sure if moving it to `check_and_call_adjust_trade_position()` will improve that - as that would allow to catch this behavior in both entries and exits in one place - BUT at the expense of not knowing the price.

Reply:
If we keep this check here we will do the same for exit you are right.

I think there are other placement possibilities, but as you said it is here that we have access to the requested price.
It is not a variable that we use yet but I think there is a high chance of it being useful in the future or even now if we are not satisfied with the current behavior.

So for the moment I vote to keep handling of similar order in execute_(entry|exit). If it is a fundamental issue for you I can move it to `check_and_call_adjust_trade_position()`

Reply:
well I'm not sure 😆 

if we keep it "here" - we should consider price.
if we move it to `check_and_call_adjust_trade_position()` - i think it's safe to ignore price - as it's really only the order adjustment part that we do want to track and prevent.

Reply:
Let's keep it there, I'll add the price in consideration 👍🏽